### PR TITLE
Fix various prometheus deploy issues

### PR DESCRIPTION
**Pull request by msherman64**

### CODE REVIEW

File: ansible/roles/prometheus/defaults/main.yml [hunk 1]
```diff
@@ -178,6 +178,7 @@ prometheus_snmp_exporter_modules: []
 # IPMI
 ####################
 
+prometheus_ipmi_exporter_modules: []
```
Comment (diurnalist) [inline ansible/roles/prometheus/defaults/main.yml:181]: This is already declared 2 lines below, isn't it?

File: ansible/roles/prometheus/tasks/config.yml [hunk 2]
```diff
@@ -191,7 +190,7 @@
     service: "{{ prometheus_services['prometheus-snmp-exporter'] }}"
   command: >
     docker run --rm \
-      --mount type=bind,src={{ node_config_directory }}/prometheus-snmp-exporter,dst=/etc/prometheus \
+      --mount type=bind,src={{ node_config_directory }}/prometheus-snmp-exporter,dst=/opt \
```
Comment (diurnalist) [inline ansible/roles/prometheus/tasks/config.yml:193]: What is the reason for this? IPMI exporter looks here now?

**Review by diurnalist — APPROVED** [review]
I think only one of these changes may be necessary

File: ansible/roles/prometheus/tasks/config.yml (hunk 2 above)
Comment (msherman64) [reply ansible/roles/prometheus/tasks/config.yml:193]: I need to revisit this post xena-upgrade, it's aged a bit. Moving it back to draft for now.

**Pull request closed without merging**